packages/trading/src/tradingSdk.ts

```diff
@@ -109,14 +110,18 @@ export class TradingSdk {
     params: WithPartialTraderParams<TradeParameters>,
     advancedSettings?: SwapAdvancedSettings,
   ): Promise<OrderPostingResult> {
-    return postSwapOrder(this.mergeParams(params), advancedSettings, this.options.orderBookApi)
+    const mergedParams = this.mergeParams(params)
+    validateSwapParameters(mergedParams)
```

Comment:
`validateSwapParameters` is already called inside of `postSwapOrder`, so we don't need to call it here as well